articles/update-manager/includes/pre-post-prerequisites.md

```diff
@@ -42,4 +48,4 @@ ms.date: 12/07/2023
              $UAMI.ClientId
     ```
   ---
-3. Import the `Az.ResourceGraph` module, ensure the module is updated to ThreadJob with the module version 2.0.3.
\ No newline at end of file
+3. Import the `Az.ResourceGraph` module, ensure the module is updated to ThreadJob with the module version 2.0.3.
```

Comment:
actually this would be better code to use.  
#Install the Resource Graph module from PowerShell Gallery
if (-not (Get-Module -Name Az.ResourceGraph -ListAvailable)) {
    Install-Module -Name Az.ResourceGraph -Force
}